docs/user-guide/index.rst

```diff
@@ -36,6 +36,9 @@
 :ref:`Next Steps <data-curator-next-steps>`
    Now that you've curated your data, let's discuss where to go next in the NeMo Framework to put it to good use.
 
+`NeMo Curator on AWS <https://github.com/NVIDIA/NeMo-Curator/tree/main/docs/user-guide/aws-examples/>`__
```

Comment:
Can you follow the pattern of other elements in the list and do something like:
```
:ref:`NeMo Curator on AWS <data-curator-aws-examples>`
```

Then create an `aws-examples/index.rst` with something like this:

```
.. _data-curator-aws-examples:

==================
AWS
==================

.. toctree::
   :maxdepth: 4
   :titlesonly:

  eks.rst
```